packages/editor/src/components/post-card-panel/index.js

```diff
@@ -130,16 +123,21 @@ export default function PostCardPanel( { className, actions } ) {
 const AVERAGE_READING_RATE = 189;
 
 // This component renders the wordcount and reading time for the post.
-function PostContentInfo( { postContent } ) {
+function PostContentInfo() {
+	const postContent = useSelect(
+		( select ) => select( editorStore ).getEditedPostAttribute( 'content' ),
+		[]
+	);
 	/*
 	 * translators: If your word count is based on single characters (e.g. East Asian characters),
 	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
 	 * Do not translate into your own language.
 	 */
 	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
-	const wordsCounted = postContent
-		? wordCount( postContent, wordCountType )
-		: 0;
+	const wordsCounted = useMemo(
+		() => ( postContent ? wordCount( postContent, wordCountType ) : 0 ),
+		[ postContent, wordCountType ]
+	);
```

Comment:
I guess you can't have the post panel open while you're typing in a block, right?

Reply:
I think you can if you can if you manually switch to the panel.

Reply:
You can if you have selected a block and then click on the `post` tab and then click again in a block and start typing..

Reply:
I'll try adding the `isTyping` value in the checks.

Reply:
It's actually interesting to see the word count change as you type, but it could be deferred if needed.

Reply:
I pushed to only update when not typing. It might be useful especially for large posts, not to recalculate on every change..

Reply:
Reverted the `isTyping` check. Let's land and revisit based on the perf results.